packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol

```diff
@@ -26,6 +24,16 @@ contract L2OutputOracle is OwnableUpgradeable, Semver {
      */
```

Comment:
Is `L2_BLOCK_TIME` safe to change? I don't think so

Reply:
It is not safe to change. I made a note of it in the other PR.